babel.config.js

```diff
@@ -191,6 +191,11 @@ module.exports = function (api) {
         test: ["packages/babel-generator"].map(normalize),
         plugins: ["babel-plugin-transform-charcodes"],
       },
+      {
+        test: ["packages/babel-register/**/*.js"].map(normalize),
+        sourceType: "script",
+        parserOpts: { allowReturnOutsideFunction: true },
```

Comment:
Is this option only needed by `packages/babel-register/src/experimental-worker.js`? If so can we modify the source so that we don't need to universally apply `allowReturnOutsideFunction`?